src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -611,4 +611,48 @@ private boolean isValidUserDirection(
             notificationGenerator
         );
     }
+
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> ftpaApplicationDecisionRefusedOrNotAdmittedNotificationHandler(
+        @Qualifier("ftpaApplicationDecisionRefusedOrNotAdmittedNotificationGenerator") List<NotificationGenerator> notificationGenerators) {
+
+        return new NotificationHandler(
+            (callbackStage, callback) -> {
+
+                AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+                boolean isNegative = asylumCase
+                    .read(AsylumCaseDefinition.FTPA_APPELLANT_DECISION_OUTCOME_TYPE, FtpaAppellantDecisionOutcomeType.class)
```

Comment:
There is also `FTPA_RESPONDENT_DECISION_OUTCOME_TYPE` field. Do we want to have notifications for that decision too?

Reply:
Fixed